### PR TITLE
client: Added a gRPC server/client interface

### PR DESCRIPTION
**Pull request by adamhicks**

### CODE REVIEW

**Review by tom-knoetze — APPROVED** [review]
nice

File: player/api.go [hunk 1]
```diff
@@ -1 +1,12 @@
...
+
+import (
+	"context"
+
+	"github.com/luno/reflex"
+)
+
+type Client interface {
+	Stream(ctx context.Context, after string, opts ...reflex.StreamOption) (reflex.StreamClient, error)
+	GetRoundParts(ctx context.Context, roundID int64) (RoundInfo, error)
```
Comment (wesrobin-luno) [inline player/api.go]: I think this should include the playerId of the player making the request

**Review by wesrobin-luno — CHANGES_REQUESTED** [review]
I just need the playerId to retrieve the parts from round_parts

File: player/api.go (hunk 1 above)
Comment (adamhicks) [reply player/api.go]: ok sure

**Review by wesrobin-luno — APPROVED** [review]

**Pull request merged**